relay-general/src/protocol/measurements.rs

```diff
@@ -11,17 +12,21 @@ pub struct Measurement {
 
 /// A map of observed measurement values.
 ///
-/// Measurements are only available on transactions. They contain measurement values of observed
-/// values such as Largest Contentful Paint (LCP).
+/// They contain measurement values of observed values such as Largest Contentful Paint (LCP).
 #[derive(Clone, Debug, Default, PartialEq, Empty, ToValue, ProcessValue)]
-#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
 pub struct Measurements(pub Object<Measurement>);
 
+impl Measurements {
+    pub fn take(self) -> Object<Measurement> {
```

Comment:
`take` usually takes a mutable reference and leaves an emptied container back. In this case, the convention is to call it `into_inner`:

```suggestion
    /// Returns the underlying object of measurements.
    pub fn into_inner(self) -> Object<Measurement> {
```